app/src/main/java/org/desarrolladorslp/teckersapp/ui/parentTeckers/ParentTeckersFragment.kt

```diff
@@ -0,0 +1,66 @@
+package org.desarrolladorslp.teckersapp.ui.parentTeckers
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import org.desarrolladorslp.teckersapp.R
+import org.desarrolladorslp.teckersapp.model.Tecker
+import org.desarrolladorslp.teckersapp.ui.deliverables.DeliverableFragment
+import org.desarrolladorslp.teckersapp.ui.teckers.TeckerViewModel
+
+class ParentTeckersFragment : Fragment(),ParentTeckerAdapter.OnHeadlineSelectedListener{
+
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var viewManager: RecyclerView.LayoutManager
+    private lateinit var teckersViewModel: TeckerViewModel
+    private lateinit var viewAdapter: RecyclerView.Adapter<ParentTeckerAdapter.TeckerHolder>
```

Comment:
From all these properties, I think the only one that needs to be here is `teckersViewModel`. Use local properties for `viewManager` and `viewAdapter`, and use Kotlin synthetic properties for `recyclerView` as I stated below.